.coveragerc

```diff
@@ -9,3 +9,4 @@ source =
 
 omit =
     *toil_container/__main__.py
+    *toil_container/lsf.py
```

Comment:
jmm. Yeah it makes sense as we can't test `lsf` in travis. Maybe there's a way, but it's not worth it. 
So LSF tests should be run in the cluster